docs/components/alerts/snippets/alerts.hbs

```diff
@@ -0,0 +1,19 @@
+---
+title: Alerts
+layout: demo.hbs
+---
+<div class="alert alert-success" role="alert">
+  <strong>Well done!</strong> You successfully read this important alert message.
+</div>
+<div class="alert alert-info" role="alert">
+  <strong>Heads up!</strong> This alert needs your attention, but it's not super important.
+</div>
+<div class="alert alert-warning alert-dismissible" role="alert">
+  <strong>Warning!</strong> Pushing the x on the right will remove this alert. For EVER!
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
+<div class="alert alert-danger" role="alert">
+  <strong>Danger!</strong> Are you sure you want to release the raptors?
+</div>
```

Comment:
I think that bit of code should be clear enough for anybody.

![image](https://cloud.githubusercontent.com/assets/7779684/20605093/d95ed548-b269-11e6-87f1-6a75612be19c.png)